### PR TITLE
Fix Alexa smart home supported platforms again

### PR DESCRIPTION
**Pull request by IceBotYT**
## Proposed change

Fix the supported platforms from being another table of contents to just the supported platforms.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by jbouwh — CHANGES_REQUESTED** [review]
This part is auto generated and it is on at the top at the page as well. This part should be removed instead. It does not need a header.

File: source/_integrations/alexa.smart_home.markdown [hunk 1]
```diff
@@ -466,77 +466,64 @@ Home Assistant supports the following integrations through Alexa using a Smart H
...
+  - [Routines](#routines-2)
+- [Timer](#timer)
+- [Vacuum](#vacuum)
+- [Valve](#valve)
+  - [Open/Close](#openclose)
+  - [Set Valve Position](#set-valve-position)
+  - [Stop the Valve](#stop-the-valve)
+- [Water heater](#water-heater)
+  - [Set target temperature](#set-target-temperature)
+  - [Operation Mode](#operation-mode)
```
Comment (jbouwh) [inline source/_integrations/alexa.smart_home.markdown:526]:
```suggestion
```

**Comment by jbouwh** [comment]
Alternative approach for this PR:
https://github.com/home-assistant/home-assistant.io/pull/30831

It completely removes the redundant table of contents.

**Comment by c0ffeeca7** [comment]
Going to close this PR as completed by #30831.

**Pull request closed without merging**